main/tests/server/src/com/google/refine/importers/XmlImporterTests.java

```diff
@@ -422,23 +421,14 @@ public static String getSampleWithTreeStructure() {
         return sb.toString();
     }
 
-    private void RunTest(String testString) {
+    private void RunTest(String testString) throws Exception {
         RunTest(testString, getOptions(job, SUT));
     }
 
-    private void RunTest(String testString, ObjectNode objectNode) {
-        try {
-            inputStream = new ByteArrayInputStream(testString.getBytes("UTF-8"));
-        } catch (UnsupportedEncodingException e1) {
-            Assert.fail();
-        }
+    private void RunTest(String testString, ObjectNode objectNode) throws Exception {
+        inputStream = new ByteArrayInputStream(testString.getBytes("UTF-8"));
```

Comment:
```suggestion
        inputStream = new ByteArrayInputStream(testString.getBytes(StandardCharsets.UTF_8));
```
Same pattern as JSON importer. Fix this and then undo all the added `throws` clauses up the call stack